tests/data/timos_hc.json

```diff
@@ -8273,48 +8273,6 @@
     "wireless": {
         "discovery": {
             "wireless_sensors": [
-                {
```

Comment:
@rsys-dev It seems that this change of MIB currently breaks some wireless sensors. Could you have a look there ? 
That does not explain why tests and save-test-data don't agree on how to regenerate the tests, but that must be fixed anyway.

Reply:
Will check this

Reply:
Sorry for the delay, had not much time the last two weeks, will have a look at it this week